app/views/shared/_in-person-verification-results-email-lower.html.erb

```diff
@@ -33,7 +33,8 @@
   <%= t(
         'user_mailer.in_person_verified.warning_contact_us_html',
         contact_us_url: MarketingSite.contact_url,
-        sign_in_url: root_url,
+        reset_password_url: forgot_password_url,
```

Comment:
Looks like [`ForgotPasswordController#show` will check for a session value which will always result in a redirect](https://github.com/18F/identity-idp/blob/2d40717b2b9258d0849f56abd10404ed54ae1387/app/controllers/forgot_password_controller.rb#L4), so maybe we can skip directly to that destination?

```suggestion
        reset_password_url: new_user_password_url,
```